Mindatarate maxpower

sethkf opened the pull request:
Made changes as requested. Tested and works as intended. A quick lookover is still recommended though.

bjhufstetler reviewed (changes requested):
1. The html needs to update to the red font when a bad value is entered for data rate.
2. The min data rate should be 0.1 MHz.

sethkf commented:
Updates have been added as requested. I changed min to 0.1 and added a min data rate check for the class name on the input box for data rate in transmitter.tsx.
